xv83.md

```diff
@@ -71,7 +81,7 @@ most of the compute allocation for ACS would be associated with `xv83`.
 The `ia39` project could be (it is currently unused) a place to share data among researchers working on the ACS project.
```

Comment:
"researchers working on the ACS project" -> "researchers working with ACS-related material"

This allows us to capture community use of pre-release ACS-related data, for example model validation and verification that is required prior to publication regardless of whether that process is part of ACS (for example with ACCESS CMIP6, field specialists required access to the 'authoritative' data for verification before it could be published as part of NESP, but the specialists were not necessarily NESP-affiliated researchers, I'm trying to capture that).

We also want to capture that ACS may provide a service to the Australian climate community through management of replica datasets that are not otherwise available at NCI but may be used by, say, Clex researchers as well as ACS researchers.

Reply:
Agreed